java/src/main/java/ola/OlaClient.java

```diff
@@ -130,6 +142,38 @@ public PluginDescriptionReply getPluginDescription(int pluginId) {
     }
 
 
+    /**
+     * Return the state for a plugin.
+     *
+     * @param pluginId number of the plugin for which to receive the state
+     * @return The list of plugings.
+     */
+    public PluginStateReply getPluginState(int pluginId) {
+        PluginStateRequest request = Ola.PluginStateRequest.newBuilder()
+                .setPluginId(pluginId)
+                .build();
+
+        return (PluginStateReply) callRpcMethod("GetPluginState", request);
+    }
+
+
+    /**
+     * Change the state of plugins.
+     *
+     * @param pluginId number of the plugin for which to change the state
+     * @param enabled  whether the plugin should be enabled or not
+     * @return Acknowledgement.
+     */
+    public Ack setPluginState(int pluginId, boolean enabled) {
+        PluginStateChangeRequest request = Ola.PluginStateChangeRequest.newBuilder()
+                .setPluginId(pluginId)
+                .setEnabled(enabled)
+                .build();
+
+        return (Ack) callRpcMethod("SetPluginState", request);
```

Comment:
I think the behaviour of patchPort is more useful:
https://github.com/OpenLightingProject/ola/blob/c4adb5034feddeaea32cb4554ef00837dbbcbec9/java/src/main/java/ola/OlaClient.java#L240
https://github.com/OpenLightingProject/ola/blob/c4adb5034feddeaea32cb4554ef00837dbbcbec9/java/src/main/java/ola/OlaClient.java#L252

Reply:
Fixed with commit b5be0b94de51153921ab8c051f000487f9d3db9f